dex-it/src/test/scala/com/wavesplatform/it/sync/PostgresHistoryDatabaseTestSuite.scala

```diff
@@ -210,6 +212,17 @@ class PostgresHistoryDatabaseTestSuite extends MatcherSuiteBase with HasPostgres
     ctx.run(querySchema[EventRecord]("events").delete)
   }
 
+  "Node should provide its name to postgres" in {
+
+    val buyOrder = mkOrder(bob, wavesBtcPair, BUY, 270477189L, 28259L)
+    placeAndAwaitAtDex(buyOrder)
+
+    findUserWithApplicationName().value shouldBe PgStatActivity(customUser, customAppName)
+
+    Seq(alice, bob).foreach(dex1.api.cancelAll(_))
```

Comment:
we have placed only one order, there is no need to use the cycle i guess

Reply:
Yes, fixed it